src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Point.java

```diff
@@ -0,0 +1,34 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+import com.rakovets.course.java.core.util.NumberUtil;
+
+public class Point {
+    private int x;
+    private int y;
+
+    public Point(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public int getX() {
+        return x;
+    }
+
+    public void setX(int x) {
+        this.x = x;
+    }
+    public int getY() {
+        return y;
+    }
+
+    public void setY(int y) {
+        this.y = y;
+    }
```

Comment:
"Violation: Getters and setters should be placed at the very end
Solution: Move getters and setters to the end"